### ISSUE TITLE
Consider putting a FAQ in the docs?

### ISSUE THREAD

**Issue by L1lith** [melonjs/melonJS#1108]
It might make it easier for people to solve issues on their own and for us to track (and post solutions to) issues as they arise

**Comment by ShukantPal** [comment]
@obiot 
webdoc has a concept of tutorials from JSDoc. You can also call them guides or wikis. You can see how to set it up here: https://www.webdoclabs.com/guides/getting-started/tutorials.html

**Comment by obiot** [comment]
oh that's 2 times interesting :
1. we can indeed have a FAQ "only" repository, where it's easier for user to contribute, and have it totally integrated with the documentation (in terms of visual / layout)
2. we can actually have the tutorial rewritten the same way and also fit within the documentation page

where are those links then available ? In the header beside the API link ?

**Comment by ShukantPal** [comment]
Yeah, the header will have two options "API" and "FAQ" (if you rename tutorials to FAQ).

If you want to have a separate repository, I would recommend using git submodules to import them into this repo so webdoc can parse the API and the tutorials in a single run.

**Comment by L1lith** [comment]
https://melonjs.org/wiki/Frequently-Asked-Questions/
The styling isn't done yet, but we're in the process of moving the wiki to the main website 😄
